.github/workflows/format-check.yml

```diff
@@ -66,6 +66,7 @@ jobs:
             Svc/FprimeFramer
             Svc/FprimeProtocol
             Svc/FprimeRouter
+            Svc/FpySequencer
```

Comment:
All of `Svc/` should now be formatted so you can squash this entire list to a single entry

```
CHECKED_DIRS: >-
            Drv
            Fw
            Os
            Svc
```